aquacomputer_d5next.c

```diff
@@ -652,12 +668,20 @@ static int aqc_read(struct device *dev, enum hwmon_sensor_types type, u32 attr,
 			*val = *val + 1;	/* Incrementing to satisfy hwmon rules */
 			break;
 		case hwmon_pwm_input:
-			ret =
-			    aqc_get_ctrl_val(priv,
+			if (priv->kind == aquaero) {
+				ret =
+					aqc_get_ctrl_val(priv,
+					    AQUAERO_CTRL_PRESET_OFFSET + channel * 2, val, 16);
```

Comment:
Extract the `2` into a macro like `AQUAERO_CTRL_PRESET_SIZE`, similar to the `AQC_TEMP_SENSOR_SIZE`, so we don't have unlabeled magic constants.